pkg/pkg.mk

```diff
@@ -9,8 +9,8 @@ PKG_BUILDDIR?=$(BINDIRBASE)/pkg/$(BOARD)/$(PKG_NAME)
 git-download: $(PKG_BUILDDIR)/.git-downloaded
 
 GIT_APPLY_PATCHES:=if test -d "$(PKG_DIR)"/patches; then \
-	git -C "$(PKG_BUILDDIR)" am --ignore-whitespace "$(PKG_DIR)"/patches/*.patch; \
-	fi
+    cd "$(PKG_BUILDDIR)" && git am --ignore-whitespace "$(PKG_DIR)"/patches/*.patch; \
+  fi
```

Comment:
no need, each individual line in a Makefile recipe runs in its own subshell instance.